pkg/api/mlflow/dao/convertors/log.go

```diff
@@ -37,5 +41,33 @@ func ConvertLogBatchRequestToDBModel(
 			RunID: runID,
 		}
 	}
-	return params, tags
+
+	metrics := make([]models.Metric, len(req.Metrics))
+	for n, metric := range req.Metrics {
+		m := models.Metric{
+			Key:       metric.Key,
+			Timestamp: metric.Timestamp,
+			Step:      metric.Step,
+			RunID:     runID,
+		}
+		if v, ok := metric.Value.(float64); ok {
+			m.Value = v
+		} else if v, ok := metric.Value.(string); ok {
```

Comment:
It would be great to have tests asserting these corner cases, and the invalid default case.